src/GameConstantsAndTypes.h

```diff
@@ -321,8 +321,6 @@ const RString& ScoreEventToString( ScoreEvent se );
 enum GameButtonType
 {
 	GameButtonType_Step,
-	GameButtonType_Fret,
-	GameButtonType_Strum,
 	GameButtonType_INVALID
```

Comment:
What is the point of keeping the enumeration if there are only steps left? How could this become invalid?